internal/pkg/cli/svc_init.go

```diff
@@ -420,6 +432,26 @@ func parseHealthCheck(df dockerfileParser) (*manifest.ContainerHealthCheck, erro
 	}, nil
 }
 
+func (o initSvcOpts) getOSArch() (operatingSystem, architecture string, err error) {
+	var os string
+	var arch string
+	if o.image != "" {
+		os = runtime.GOOS
+		arch = runtime.GOARCH
```

Comment:
Can we do something like this instead? 
```go
os, arch := "linux", "amd64"
if o.image == "" {
   os, arch, err = o.dockerEngineValidator.Platform()
}
```

Reply:
If someone on an ARM machine supplies an existing Docker image that is for ARM architecture, they won't know why it is failing, though, right?

Reply:
oh that makes sense!
then let's do:
```go
os, arch := runtime.GOOS, runtime.GOARCH
if o.image == "" {
   ...
}
```